### ISSUE TITLE
Dotnet CommandLine Tool Error on Runtime

### ISSUE THREAD

**Issue by mariohines** [RicoSuter/NSwag#3443]
When I run the commandline tool installed via `dotnet tool` it doesn't work according to the exception error with the `/runtime:<runtime>` argument. I've tried many times. After looking through the code base, I figured out why. The `ExecuteDocumentCommand` does not take that parameter as input. It uses an enumeration called `Runtime.Default` and then checks the runtime of the actual console application to test against.

Also, through more experimentation, found that you can set the runtime in the json file to `Default` and it runs without much issue, however the `nswag` executable defaults to Net 5.0 which will break if you don't have Net 5.0 installed. And their doesn't seem to be a way to point it to the correct version outside of going to the tool directory and manually running the right version (which is painful) with `dotnet dotnet-nswag.dll <command> <arguments>`.

**Comment by jeremyVignelles** [comment]
A .net dll is built for a specific runtime, and you're expected to run it with the proper runtime.
You can't run it to let it decide the runtime since you need the runtime to run it...

**Comment by mariohines** [comment]
True. But the default is Net5. And the dotnet tool, `NSwag.ConsoleCore` doesn't take `/runtime` as an argument. I did however find where that is located. The file `nswag.cmd` comes with the zip archive and the MSI installer that does take `/runtime` as an argument, but it works kinda funny as it brings up the command list and then you have to type in `run` to make it work. Bottom line, the dotnet tool does not accept `/runtime` as an argument and it's defaulted to Net5.
